.vscode/cspell.json

```diff
@@ -27,6 +27,7 @@
     "*.exe",
     "*.a",
     "*.lib",
+    "*.svg",
```

Comment:
At one point, this PR had an update to an SVG file, which has since been deleted in #6591, but this is still a good change.